README.md

```diff
@@ -222,6 +222,9 @@ let us know and we'll be happy to include it here!
 - **Python D2 diagram builder**: [https://github.com/MrBlenny/py-d2](https://github.com/MrBlenny/py-d2)
 - **Clojure D2 transpiler**: [https://github.com/judepayne/dictim](https://github.com/judepayne/dictim)
 - **JavaScript D2 diagram builder**: [https://github.com/Kreshnik/d2lang-js](https://github.com/Kreshnik/d2lang-js)
+- **Maven plugin**: [https://github.com/andrinmeier/unofficial-d2lang-confluence-plugin](https://github.com/andrinmeier/unofficial-d2lang-confluence-plugin)
```

Comment:
@alixander Seems you've made mistake in url

Reply:
indeed, was fixed in a subsequent commit (https://github.com/terrastruct/d2/commit/bb264df5ca432f946608a307b0f4ba5f47204982). ty for keeping an eye out!